packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -56,7 +62,13 @@ export function addRoutesToApp(app: Express) {
     handleWith(FetchConfirmedCountryWeeklyReportRoute),
   );
   app.get('/v1/country', handleWith(FetchCountries));
+  app.get('/v1/country/:countryCode/sites', handleWith(FetchCountrySites));
   app.get('/v1/weeklyReport/:organisationUnitCode', handleWith(FetchCountryWeeklyReportRoute));
+  app.use('/v1/weeklyReport/:organisationUnitCode/sites', useSites);
+  app.get(
+    '/v1/weeklyReport/:organisationUnitCode/sites',
+    handleWith(FetchCountryWeeklyReportRoute),
+  );
```

Comment:
```suggestion
  app.get(
    '/v1/weeklyReport/:organisationUnitCode/sites',
    useSites,
    handleWith(FetchCountryWeeklyReportRoute),
  );
```

Reply:
I prefer this pattern for adding middleware to a specific route

Reply:
Good point. It will be used in my next PR - I added the '/v1/weeklyReport/:organisationUnitCode/sites routes here but not their implementation

Reply:
Yes that's better!